### PR TITLE
[PEPPER-1143] [DSM] [Playwright] Under Age Quick Filter test

### PR DESCRIPTION
**Pull request by aweng98**
[PEPPER-1143](https://broadworkbench.atlassian.net/browse/PEPPER-1143)

New user has limited permissions.

Screenshot 2023-11-20 at 4 00 44 PM

[PEPPER-1143]: https://broadworkbench.atlassian.net/browse/PEPPER-1143?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .circleci/export-playwright-env.sh [hunk 1]
```diff
@@ -19,6 +19,12 @@ export dsmUser1Password=$(vault read --format=json secret/pepper/test/v1/e2e | j
 echo "export DSM_USER1_PASSWORD=$dsmUser1Password" >> playwright-env/envvars
 echo "export DSM_USER1_EMAIL=$dsmUser1" >> playwright-env/envvars
 
+# Additional DSM User (DSM general testing)
```
Comment (aweng98) [inline .circleci/export-playwright-env.sh:22]: New general DSM user: avoiding bug PEPPER-1031

File: playwright-e2e/tests/dsm/participant-list/saved-views-custom-columns-filter-condition.spec.ts [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+import { testWithUser2 as test } from 'fixtures/dsm-fixture';
+import { AdditionalFilter, CustomViewColumns } from 'dsm/component/filters/sections/search/search-enums';
 import { StudyEnum } from 'dsm/component/navigation/enums/selectStudyNav-enum';
 import ParticipantListPage from 'dsm/pages/participant-list-page';
 import { offsetDaysFromToday } from 'utils/date-utils';
+import crypto from 'crypto';
 
-// Test is broken until bug is fixed. https://broadworkbench.atlassian.net/browse/PEPPER-1031
-test.describe.fixme('Participants list search and filter', () => {
+test.describe('Participants List Search with Filtering Condition', () => {
```
Comment (aweng98) [inline playwright-e2e/tests/dsm/participant-list/saved-views-custom-columns-filter-condition.spec.ts:9]: Disable test is now again passing after switching to use newly added DSM user.

**Review by Rinchan20 — APPROVED** [review]
looks good - passes in dev - just needed to add the Medical Records permission (this one -> Medical Records: View, request records and tissue) - and then the dsm user can see the Sample Column options in customize view and it passes

**Comment by aweng98** [comment]
> looks good - passes in dev - just needed to add the Medical Records permission (this one -> Medical Records: View, request records and tissue) - and then the dsm user can see the Sample Column options in customize view and it passes

Are you referring to new user `DSM_USER2_EMAIL`? Do I need to update permisssions for this user on Dev and Test env?

**Comment by aweng98** [comment]
Checked and added Medical Records permisson for new user `Maya.Bonechewer@test.firecloud.org` on Dev and Test env.

**Pull request merged**